### PR TITLE
[579] leverage ES6 formatting for readability

### PR DESCRIPTION
**Pull request by sbatson5**
# What's in this PR?

This is an example of how we can de-structure properties from the Ember namespace to (hopefully) increase readability. Wanted to throw up an example for now to see if this is something people are interested in.

Would update styleguide as part of this PR.

## References
Fixes https://github.com/code-corps/code-corps-ember/issues/579

### CODE REVIEW

**Comment by JoshSmith** [comment]
This is really great @sbatson5! Let's make this happen.

File: app/components/organization-settings-form.js [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
+  flashMessages: service(),
 
   actions: {
     save() {
-      const flashMessages = Ember.get(this, 'flashMessages');
-      this.get('organization').save()
-        .then(function() {
-          flashMessages.success("Organization updated successfully");
+      const flashMessages = get(this, 'flashMessages');
+      get(this, 'organization').save().then(() => {
```
Comment (sbatson5) [inline app/components/organization-settings-form.js]: Functionally, this hasn't changed. Just thought it read better with the arrow function.

**Comment by sbatson5** [comment]
@JoshSmith Not sure how much I want to tackle in this 1 PR--already touched 25 files. It may be nice to break this up and tackle more in a future PR. Thoughts?

**Comment by JoshSmith** [comment]
@sbatson5 I agree, let's break it up and do it in more than 1 PR.

**Comment by JoshSmith** [comment]
@sbatson5 a few test failures, ping me when they're good and will approve.

**Comment by JoshSmith** [comment]
🙌

**Pull request merged**